eng/scripts/test-matrix/test-matrix-functions.ps1

```diff
@@ -0,0 +1,317 @@
+Set-StrictMode -Version "4.0"
+
+function CreateDisplayName([string]$parameter, [hashtable]$displayNames) {
+    $name = $parameter
+
+    if ($displayNames[$parameter]) {
```

Comment:
I thought this actually failed if the key didn't exist. I usually do a check similar to: 
```suggestion
    if ($displayNames.ContainsKey($parameter)) {
```

Reply:
huh, I thought I had changed this. Weird that it's not failing with strict mode. I'll look into it.